tests/scene/resources/test_image_texture.h

```diff
@@ -0,0 +1,225 @@
+/*                                                                        */
+/* Permission is hereby granted, free of charge, to any person obtaining  */
+/* a copy of this software and associated documentation files (the        */
+/* "Software"), to deal in the Software without restriction, including    */
+/* without limitation the rights to use, copy, modify, merge, publish,    */
+/* distribute, sublicense, and/or sell copies of the Software, and to     */
+/* permit persons to whom the Software is furnished to do so, subject to  */
+/* the following conditions:                                              */
+/*                                                                        */
+/* The above copyright notice and this permission notice shall be         */
+/* included in all copies or substantial portions of the Software.        */
+/*                                                                        */
+/* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,        */
+/* EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF     */
+/* MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. */
+/* IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY   */
+/* CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,   */
+/* TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE      */
+/* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                 */
+/**************************************************************************/
+
+#ifndef TEST_IMAGE_TEXTURE_H
+#define TEST_IMAGE_TEXTURE_H
+
+#include "scene/resources/image_texture.h"
+
+#include "tests/test_macros.h"
+#include "tests/test_utils.h"
+
+namespace TestImageTexture {
+const int default_height = 64;
+const int default_width = 64;
+const int default_channels = 3;
+const Image::Format default_format = Image::Format::FORMAT_RGB8;
+
+static Ref<Image> create_test_image_base(int p_channels, Image::Format p_format) {
+	Vector<uint8_t> data;
+	data.resize(default_width * default_height * p_channels);
+
+	// This loop fills the data with image pixel values (RGBA format).
+	for (int y = 0; y < default_height; y++) {
+		for (int x = 0; x < default_width; x++) {
+			int offset = (y * default_width + x) * p_channels;
+			for (int c = 0; c < p_channels; c++) {
+				data.set(offset + c, 255);
+			}
+		}
+	}
+
+	return Image::create_from_data(default_width, default_height, false, p_format, data);
+}
+
+static Ref<Image> create_test_image() {
+	return create_test_image_base(default_channels, default_format);
+}
+
+static Ref<Image> create_test_image_with_alpha() {
+	return create_test_image_base(4, Image::FORMAT_RGBA8);
+}
+
+TEST_CASE("[ImageTexture] Constructor") {
+	// Create an ImageTexture object.
+	Ref<ImageTexture> imageTexture;
+	imageTexture.instantiate();
+
+	// Ensure that it's not null.
+	REQUIRE(imageTexture.is_valid());
```

Comment:
Remove comments that just state what the code is doing below (where it's obvious):

```suggestion
	Ref<ImageTexture> imageTexture;
	imageTexture.instantiate();

	REQUIRE(imageTexture.is_valid());
```